### PR TITLE
Make connection security policy string lowercase for prometheus

### PR DESCRIPTION
**Pull request by gargnupur**
Signed-off-by: gargnupur <gargnupur@google.com>

**What this PR does / why we need it**:
Make connection security policy string lowercase for prometheus
We need this to be compatible to Istio 1.4

### CODE REVIEW

File: extensions/stats/plugin.cc [hunk 1]
```diff
@@ -97,8 +105,8 @@ void map_request(IstioDimensions& instance,
   instance[request_protocol] = request.request_protocol;
   instance[response_code] = std::to_string(request.response_code);
   instance[response_flags] = request.response_flag;
-  instance[connection_security_policy] = std::string(
-      ::Wasm::Common::AuthenticationPolicyString(request.service_auth_policy));
+  instance[connection_security_policy] = to_lower(std::string(
```
Comment (bianpengyuan) [inline extensions/stats/plugin.cc]: How about using this https://github.com/abseil/abseil-cpp/blob/master/absl/strings/ascii.h#L168？

Comment (gargnupur) [reply extensions/stats/plugin.cc]: I thought we were not using absl stuff for wasm pluging?

Comment (bianpengyuan) [reply extensions/stats/plugin.cc]: String related utils should be fine. they do not involve system calls.

**Pull request merged**